.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: 
+  false
+
+language: 
+  c
```

Comment:
`language: c` uses a very lightweight image. I've been using it as a default but maybe there are other good practices now